projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "block.electrodynamics.mineralwasher" : "洗矿机",
+  "block.electrodynamics.chemicalmixer" : "化学混合器",
+  "block.electrodynamics.chemicalcrystallizer" : "化学结晶器",
+  "block.electrodynamics.circuitbreaker" : "断路器",
+  "block.electrodynamics.multimeterblock" : "万用表",
+  "block.electrodynamics.lathe" : "车床",
+  "block.electrodynamics.reinforcedalloyer" : "强化合金熔炼炉",
+  "block.electrodynamics.cobblestonegenerator" : "圆石温差发电机",
+  "block.electrodynamics.creativepowersource" : "创造能源",
+  "block.electrodynamics.chargerlv" : "120 V 充电器",
+  "block.electrodynamics.chargermv" : "240 V 充电器",
+  "block.electrodynamics.chargerhv" : "480 V 充电器",
+  "block.electrodynamics.tanksteel" : "钢流体储罐",
+  "block.electrodynamics.tankreinforced" : "加固流体储罐",
+  "block.electrodynamics.tankhsla" : "高强度钢流体储罐",
+  "block.electrodynamics.creativefluidsource" : "创造流体源",
+  "block.electrodynamics.fluidvoid" : "流体销毁机",
+  "block.electrodynamics.electrolyticseparator" : "电解分离机",
+  "block.electrodynamics.wiretin" : "锡导线",
+  "block.electrodynamics.wireiron" : "铁导线",
+  "block.electrodynamics.wirecopper" : "铜导线",
+  "block.electrodynamics.wiresilver" : "银导线",
+  "block.electrodynamics.wiregold" : "金导线",
+  "block.electrodynamics.wiresuperconductive" : "超导导线",
+  "block.electrodynamics.wireinsulatedtin" : "绝缘锡导线",
+  "block.electrodynamics.wireinsulatediron" : "绝缘铁导线",
+  "block.electrodynamics.wireinsulatedcopper" : "绝缘铜导线",
+  "block.electrodynamics.wireinsulatedsilver" : "绝缘银导线",
+  "block.electrodynamics.wireinsulatedgold" : "绝缘金导线",
+  "block.electrodynamics.wireinsulatedsuperconductive" : "绝缘超导导线",
+  "block.electrodynamics.wirehighlyinsulatedtin" : "高度绝缘锡导线",
+  "block.electrodynamics.wirehighlyinsulatediron" : "高度绝缘铁导线",
+  "block.electrodynamics.wirehighlyinsulatedcopper" : "高度绝缘铜导线",
+  "block.electrodynamics.wirehighlyinsulatedsilver" : "高度绝缘银导线",
+  "block.electrodynamics.wirehighlyinsulatedgold" : "高度绝缘金导线",
+  "block.electrodynamics.wirehighlyinsulatedsuperconductive" : "高度绝缘超导导线",
+  "block.electrodynamics.wireceramicinsulatedtin" : "陶瓷绝缘锡导线",
+  "block.electrodynamics.wireceramicinsulatediron" : "陶瓷绝缘铁导线",
+  "block.electrodynamics.wireceramicinsulatedcopper" : "陶瓷绝缘铜导线",
+  "block.electrodynamics.wireceramicinsulatedsilver" : "陶瓷绝缘银导线",
+  "block.electrodynamics.wireceramicinsulatedgold" : "陶瓷绝缘金导线",
+  "block.electrodynamics.wireceramicinsulatedsuperconductive" : "陶瓷绝缘超导导线",
+  "block.electrodynamics.wirelogisticstin" : "锡物流导线",
+  "block.electrodynamics.wirelogisticsiron" : "铁物流导线",
+  "block.electrodynamics.wirelogisticscopper" : "铜物流导线",
+  "block.electrodynamics.wirelogisticssilver" : "银物流导线",
+  "block.electrodynamics.wirelogisticsgold" : "金物流导线",
+  "block.electrodynamics.wirelogisticssuperconductive" : "超导物流导线",
+  "block.electrodynamics.pipesteel" : "钢导管",
+  "block.electrodynamics.multisubnode" : "多方块子节点",
+  "block.electrodynamics.energizedalloyer" : "充能合金熔炼炉",
+  "block.electrodynamics.seismicmarker" : "地质标记",
```

```suggestion
  "block.electrodynamics.seismicmarker" : "地质标记器",
```
没有在游戏内找到这个东西 但是看block的话应该是属于方块

---

建议用机

---

> 建议用机

标记机，听上去不太妙（